packages/typescript-configs/core.json

```diff
@@ -15,7 +15,7 @@
         "allowUnusedLabels": false,
         "allowUnreachableCode": false,
         "noFallthroughCasesInSwitch": true,
-        "noUnusedLocals": true,
+        "noUnusedLocals": false,
```

Comment:
You can remove the field as the default value is `false`.

Also, IMO, the same concept apply to the following field so they could also be removed:

```
```
"allowUnusedLabels": false,
"allowUnreachableCode": false,
"noFallthroughCasesInSwitch": true,
```